src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -443,3 +445,41 @@ def generate_maps(
     #   }
 
     return maps
+
+
+def subset_results_by_state(processed_results_dir: str, state: str) -> None:
+    # Loads final results and subsets those files to a provide state excluding the Social Security Observer
+
+    from vivarium_census_prl_synth_pop.constants.metadata import US_STATE_ABBRV_MAP
```

Comment:
Why did you import this in here? All import should be at the top of the file. You actually are already importing `metadata` so remove this import and replace `US_STATE_ABBRV_MAP` below with `metadata.US_STATE_ABBRV_MAP`